charts/digital-twin-bundle/values.yaml

```diff
@@ -0,0 +1,71 @@
+# #############################################################################
+# Copyright (c) 2025 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# #############################################################################
+
+# Default values for digital-twin-bundle
+# This is a YAML-formatted file.
+
+# Enable/disable the entire bundle
+enabled: true
+
+# DT Registry Configuration
+dt-registry:
+  enabled: true
+  image:
+    repository: eclipse-tractusx/digital-twin-registry
+    tag: "0.6.2"
+  service:
+    type: ClusterIP
+    port: 8080
+  resources:
+    requests:
+      memory: "512Mi"
+      cpu: "250m"
+    limits:
+      memory: "1Gi"
+      cpu: "500m"
+  config:
+    registry:
+      api:
+        baseUrl: "http://dt-registry:8080"
+      persistence:
+        type: "postgres"
+      security:
+        enabled: true
+        oauth2:
+          clientId: "dt-registry-client"
+          clientSecret: "dt-registry-secret"
+          tokenUrl: "http://identity-trust-bundle:8080/oauth/token"
```

Comment:
Is this the correct way to hardcode this paths? Could we use variables in perspective that we have to refactore this within the current working pakage?